src/sparseml/pytorch/models/registry.py

```diff
@@ -207,8 +207,11 @@ def register(
         if not isinstance(key, List):
             key = [key]
 
-        def decorator(const_func):
-            wrapped_constructor = ModelRegistry._registered_wrapper(key[0], const_func)
+        def decorator(constructor_func):
```

Comment:
confirming that the changes here are just for readability?

Reply:
yes!